### PR TITLE
Reverted Controller transformation code to resolve translation issue

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Reverted Controller transformation code to resolve translation issue

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs [hunk 1]
```diff
@@ -1512,26 +1512,19 @@ internal static float CalculateDeadzone(float a, float deadzone)
...
-
-            rotation.x = -pose.Orientation.x;
-            rotation.y = -pose.Orientation.y;
-            rotation.z = pose.Orientation.z;
-            rotation.w = pose.Orientation.w;
-
-            xrtkPose.Rotation = rotation;
-
-            return xrtkPose;
+        public static MixedRealityPose ToMixedRealityPose(this Posef pose, bool adjustForEyeHeight = false)
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/OculusApi.cs]: We probably don't have to revert this, do we?

**Comment by StephenHodgson** [comment]
azp /run

**Comment by StephenHodgson** [comment]
azp run

**Comment by SimonDarksideJ** [comment]
Hold releasing after this pr, there is a new api out this week and might as well update that too

**Comment by StephenHodgson** [comment]
Well the upm server is down anyway. Might hold up our approval process.

**Comment by StephenHodgson** [comment]
Fixed the upm server

**Pull request merged**